x/market/client/cli/txDeal.go

```diff
@@ -0,0 +1,39 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+
+)
+
+func NewCreateDealCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create-deal [flags]",
+		Short: "create a new deal",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			txf := tx.NewFactoryCLI(clientCtx, cmd.Flags()).WithTxConfig(clientCtx.TxConfig).WithAccountRetriever(clientCtx.AccountRetriever)
+
+			txf, msg, err := NewBuildCreateDealMsg(clientCtx, txf, cmd.Flags())
+			if err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxWithFactory(clientCtx, txf, msg)
+		},
+	}
+
+	cmd.Flags().AddFlagSet(FlagSetCreateDeal())
+	flags.AddTxFlagsToCmd(cmd)
+
+	_ = cmd.MarkFlagRequired(FlagDealFile)
+
```

Comment:
```suggestion
	cmd.Flags().String(FlagDealFile, "" ,"Deal json file path")
	_ = cmd.MarkFlagRequired(FlagDealFile)

	flags.AddTxFlagsToCmd(cmd)

```

How about fix it like this?
I think we don't need a `market/client/cli/flags.go` files.

Reply:
I think this review is related to below issue. So I will resolve this conversation. Thank you for your feedback 😀